app/apis/api_application.py

```diff
@@ -121,3 +122,51 @@ def patch(self, application_id:int):
         db.session.commit()
         db.session.close()
         return response_body
+
+@app_info_namespace.route('/<int:application_id>/evaluation')
+class ApiEvaluation(Resource):
+    upload_parser = reqparse.RequestParser()
+    upload_parser.add_argument('file', location='files', type=FileStorage, required=True)
+
+    @app_info_namespace.expect(upload_parser)
+    def post(self, application_id:int):
+        """update data to be evaluated"""
+        args = self.upload_parser.parse_args()
+        file = args['file']
+        eval_data_path = "eval-{0:%Y%m%d%H%M%S}.txt".format(datetime.datetime.utcnow())
+
+        sobj = Service.query.filter_by(application_id=application_id).first_or_404()
+
+        drucker_dashboard_application = DruckerDashboardClient(logger=logger, host=sobj.host)
+        response_body = drucker_dashboard_application.run_upload_evaluation_data(file, eval_data_path)
+
+        if not response_body['status']:
+            raise Exception('Failed to upload')
+        eobj = Evaluation(application_id=application_id, data_path=eval_data_path)
+        db.session.add(eobj)
+        db.session.flush()
+        evaluation_id = eobj.evaluation_id
+        db.session.commit()
+        db.session.close()
+
+        return {"status": True, "evaluation_id": evaluation_id}
```

Comment:
> Not to upload the same evaluation data, separate evaluate and upload test data

At the moment, this PR does not include the duplication check for evaluation data. If we do it, we need to add that function to `reckurd` repository.

Anyway, we can add the duplication upload test here.